Library/Formula/newlisp.rb

```diff
@@ -16,17 +11,16 @@ class Newlisp < Formula
     sha256 "1f99334ef4a7f02f046a6f748b763b266a742a838cf4079d89ed3caf5801af7d" => :mavericks
   end
 
-  devel do
-    url "http://www.newlisp.org/downloads/development/newlisp-10.6.4.tgz"
-    sha256 "1b769d8026241a02ac7e6fc326c5d9b99b976482a40a0c8d5c828df72275aa18"
-  end
-
   depends_on "readline" => :recommended
 
   def install
     # Required to use our configuration
     ENV.append_to_cflags "-DNEWCONFIG -c"
 
+    # fix the prefix in a source file
+    inreplace "guiserver/newlisp-edit.lsp", "#!/usr/local/bin/newlisp", "#!/usr/bin/env newlisp"
+    inreplace "guiserver/newlisp-edit.lsp", "/usr/local/bin/newlisp", "HOMEBREW_PREFIX/bin/newlisp"
```

Comment:
Use `#{opt_bin}` instead of `HOMEBREW_PREFIX/bin` (I assume that’s a typo and you wanted to use `#{HOMEBREW_PREFIX}`).